SFTP/node.putFile.js

```diff
@@ -0,0 +1,19 @@
+let logger = global.logger;
+
+module.exports = async (connectorData, inputData) => {
+	logger.trace("SFTP Put File Node: Invoked!");
```

Comment:
We need an info level log that says which file we are going to put

Reply:
Done!